docs/guides/runbooks_steps_microsoft_teams.md

```diff
@@ -0,0 +1,222 @@
+  integration_slug = "microsoft_teams"
+  slug             = "notify_channel"
+  type             = "incident"
+}
+
+resource "firehydrant_runbook" "microsoft_teams_notify_channel_runbook" {
+  name = "microsoft-teams-notify-channel-runbook"
+
+  steps {
+    name      = "Notify a Microsoft Teams Channel"
+    action_id = data.firehydrant_runbook_action.microsoft_teams_notify_channel.id
+
+    config = jsonencode({
+      channels = "General, Incidents"
+    })
+    
+    repeats          = true
+    repeats_duration = "PT15M"
+  } 
+}
+```
+
+### Notify Channel - Steps Argument Reference
+
+* `action_id` - (Required) The ID of the runbook action for the step.
+* `config` - (Required) JSON string representing the configuration settings for the step.
+  Use [Terraform's jsonencode](https://www.terraform.io/language/functions/jsonencode)
+  function so that [Terraform can guarantee valid JSON syntax](https://www.terraform.io/language/expressions/strings#generating-json-or-yaml).
+* `name` - (Required) The name of the step.
+* `automatic` - (Optional) Whether this step should automatically execute.
+* `repeats` - (Optional) Whether this step should repeat. Defaults to `false`.
+    When this value is `true`, `repeats_duration` _must_ be provided.
+* `repeats_duration` - (Optional) How often this step should repeat in ISO8601.
+  Example: PT10M [Format Spec](https://www.digi.com/resources/documentation/digidocs/90001437-13/reference/r_iso_8601_duration_format.htm)
+  This value _must_ be provided if `repeats` is `true`. This value _must not_ be provided if `repeats` is `false`.
+
+The `config` block supports:
+
+* `channels` - (Required) A comma separated list of channels to notify.
+
+## Notify Channel With a Custom Message
+
+The [Microsoft Teams **Notify Channel With a Custom Message** step](https://support.firehydrant.com/hc/en-us/articles/360058202811)
+allows FireHydrant users to automatically notify different channels with a custom message during an incident.
+
+### Notify Channel With a Custom Message - Example Usage
+
+Basic usage:
+```hcl
+data "firehydrant_runbook_action" "microsoft_teams_notify_channel_custom" {
+  integration_slug = "microsoft_teams"
+  slug             = "notify_channel_custom_message"
+  type             = "incident"
+}
+
+resource "firehydrant_runbook" "microsoft_teams_notify_channel_custom_runbook" {
+  name = "microsoft-teams-notify-channel-custom-runbook"
+  type = "incident"
+
+  steps {
+    name      = "Notify a Microsoft Teams Channel with a Custome Message"
+    action_id = data.firehydrant_runbook_action.microsoft_teams_notify_channel_custom.id
+
+    config = jsonencode({
+      channels = "General, Incidents"
+      message  = "Please check-in with your current status on this {{ incident.severity }} incident\n\n```\n/firehydrant add note I'm calculating the power required by the flux capacitor\n```\n"
+    })
+  }
+
+  repeats   = false
+  automatic = true
+}
```

Comment:
move repeats and automatic into the step block, fix typo
```suggestion
  steps {
    name      = "Notify a Microsoft Teams Channel with a Custom Message"
    action_id = data.firehydrant_runbook_action.microsoft_teams_notify_channel_custom.id

    config = jsonencode({
      channels = "General, Incidents"
      message  = "Please check-in with your current status on this {{ incident.severity }} incident\n\n```\n/firehydrant add note I'm calculating the power required by the flux capacitor\n```\n"
    })
    
    repeats   = false
    automatic = true
  }
}
```